kano_world/share_helpers.py

```diff
@@ -27,7 +29,7 @@ def login_and_share(file_path, title, app_name):
     success, unused = login_using_token()
 
     if not success:
-        run_cmd('kano-login -r')
+        run_cmd('kano-login -r', localised=True)
```

Comment:
Perfect - Just a note for a more generic behaviour in the future.
I think we can assume `localised=True` by default, and only force `LC_ALL="C"` on its absence.

 * https://github.com/KanoComputing/kano-toolset/blob/master/kano/utils/shell.py#L28

Reply:
That's true. I'm definitely up for that, but I think we've never tried it given the huge score of `run_cmd` in the system.